Microsoft.Toolkit.Parsers/Markdown/Blocks/ListBlock.cs

```diff
@@ -169,8 +169,8 @@ internal static ListBlock Parse(string markdown, int start, int maxEnd, int quot
                         }
                         else
                         {
-                            // Inline text.
-                            AppendTextToListItem(currentListItem, markdown, lineInfo.FirstNonWhitespaceChar, lineInfo.EndOfLine, true);
+                            // Inline text. Ignores the 4 spaces that are used to continue the list.
+                            AppendTextToListItem(currentListItem, markdown, lineInfo.StartOfLine + 4, lineInfo.EndOfLine, true);
```

Comment:
@paulbartrum This is a bug fix for where a ListBlockItem will swallow any additional formatting Whitespace, such as used in CodeBlocks.

Reply:
Turns out it was a little bit more complicated then that, added more workaround in the next commit.